style.css

```
@@ -0,0 +1,33 @@
+body {
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+  margin: 60px;
+}
+
+table {
+  margin-left: auto;
+  margin-right: auto;
+  border-style: dotted;
+  border-width: 2px;
+  border-collapse: collapse;
+  margin-bottom: 50px;
+}
+
+th {
+  border-style: solid;
+  border-width: 2px;
+  width: 125px;
+  padding: 5px;
+}
+
+td {
+  border-style: dotted;
+  border-width: 2px;
+}
+
+p {
+  width: 550px;
+  margin-left: auto;
+  margin-right: auto;
+}
```

End files with an empty return to appease git.
```suggestion
}

```